[2023-08-31T03:53:10.576+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_sale_data.validate_result manual__2023-08-31T03:53:03.900971+00:00 [queued]>
[2023-08-31T03:53:10.586+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_sale_data.validate_result manual__2023-08-31T03:53:03.900971+00:00 [queued]>
[2023-08-31T03:53:10.587+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-31T03:53:10.599+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): validate_result> on 2023-08-31 03:53:03.900971+00:00
[2023-08-31T03:53:10.605+0000] {standard_task_runner.py:57} INFO - Started process 5686 to run task
[2023-08-31T03:53:10.607+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'process_sale_data', 'validate_result', 'manual__2023-08-31T03:53:03.900971+00:00', '--job-id', '166', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpfvumb3j8']
[2023-08-31T03:53:10.609+0000] {standard_task_runner.py:85} INFO - Job 166: Subtask validate_result
[2023-08-31T03:53:10.651+0000] {task_command.py:410} INFO - Running <TaskInstance: process_sale_data.validate_result manual__2023-08-31T03:53:03.900971+00:00 [running]> on host 888526fa4a36
[2023-08-31T03:53:10.729+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_sale_data' AIRFLOW_CTX_TASK_ID='validate_result' AIRFLOW_CTX_EXECUTION_DATE='2023-08-31T03:53:03.900971+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-31T03:53:03.900971+00:00'
[2023-08-31T03:53:10.864+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-08-31T03:53:10.869+0000] {sql.py:375} INFO - Running statement: SELECT COUNT(*) FROM consumption_alcoholic_20230831, parameters: None
[2023-08-31T03:53:10.870+0000] {sql.py:384} INFO - Rows affected: 1
[2023-08-31T03:53:10.875+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-08-31T03:53:10.878+0000] {sql.py:375} INFO - Running statement: SELECT COUNT(*) FROM consumption_cereals_bakery_20230831, parameters: None
[2023-08-31T03:53:10.879+0000] {sql.py:384} INFO - Rows affected: 1
[2023-08-31T03:53:10.886+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-08-31T03:53:10.889+0000] {sql.py:375} INFO - Running statement: SELECT COUNT(*) FROM consumption_meats_poultry_20230831, parameters: None
[2023-08-31T03:53:10.890+0000] {sql.py:384} INFO - Rows affected: 1
[2023-08-31T03:53:10.916+0000] {logging_mixin.py:149} INFO - Import data successful
[2023-08-31T03:53:10.917+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-08-31T03:53:10.926+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=process_sale_data, task_id=validate_result, execution_date=20230831T035303, start_date=20230831T035310, end_date=20230831T035310
[2023-08-31T03:53:10.980+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-31T03:53:11.004+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
